package.json

```diff
@@ -50,6 +50,7 @@
   ],
   "dependencies": {
     "dotenv": "^8.2.0",
+    "emotion-theming": "^10.0.27",
```

Comment:
I Just a minor tweak. `emotion-theming` is only used in the client part of your application (= React). Maybe this dependency can be moved to `client/package.json`

Reply:
Done, thanks!